docs/components/modeler/desktop-modeler/element-templates/element-templates.md

```diff
@@ -23,4 +23,4 @@ As seen in the _Mail Task_ example above, Modeler allows properties of custom el
 
 Refer to the following resources to learn more about element templates:
 
-- [Element template documentation](https://github.com/camunda/desktop-modeler/tree/master/docs/element-templates)
+- [Element template documentation](/components/modeler/desktop-modeler/element-templates/element-templates.md)
```

Comment:
I confused myself and you. I believe the intention is for `/element-templates/` to link to `/about-templates/`. Clearly should have separated the PRs 🙈

Reply:
Sorry for the back and forth, but no, I _do_ think this is linking to itself. At least it is when I run locally. 

I don't see a document with the ID `element-templates.md` explicitly set, so I'm actually surprised to see docusaurus resolve it at all -- though it looks like it is just resolving it by source path instead of ID. 

So back to my original point, I think this link is incorrect 😅

Reply:
Can you update it to what Nico originally had in his first commit? It should link to `/about-templates/`.